sqlglot/generator.py

```diff
@@ -1226,7 +1220,26 @@ def index_sql(self, expression: exp.Index) -> str:
         include = self.expressions(expression, key="include", flat=True)
         if include:
             include = f" INCLUDE ({include})"
-        return f"{unique}{primary}{amp}{index}{name}{table}{using}{columns}{include}{partition_by}{where}"
+        with_storage = self.expressions(expression, "with_storage", flat=True)
```

Comment:
Minor one, mostly for consistency with the rest of `self.expressions` calls.
```suggestion
        with_storage = self.expressions(expression, key="with_storage", flat=True)
```